### PR TITLE
Update

### PR DESCRIPTION
**Pull request by mathieu-aubin**
Make use of heredocs for creating ini files and xargs for restarting services

### CODE REVIEW

File: scripts/blackfire.sh [hunk 1]
```diff
@@ -1,26 +1,25 @@
...
-"
+__EOF__
 
-echo "$agent" > "/etc/blackfire/agent"
-echo "$client" > "/home/vagrant/.blackfire.ini"
-
-service php7.1-fpm restart
-service blackfire-agent restart
+echo "php7.1-fpm blackfire-agent" | \
+xargs -n1 bash -c 'service ${0} restart; sleep 0.1'
```
Comment (andrewhood125) [inline scripts/blackfire.sh]: This is taking DRY to the super extreme. Fascinating approach but would strongly advise against it.

**Comment by andrewhood125** [comment]
I'm totally in favor of the heredocs.

**Comment by svpernova09** [comment]
I would rather sacrifice DRY for readability

--
- Joe Ferguson
JoeFerguson.me
MemphisPHP.org

**Comment by svpernova09** [comment]
Thanks for taking the time to PR. Going to hold off at the moment.

**Comment by mathieu-aubin** [comment]
I'm glad it got you guys talking hehe..

**Pull request closed without merging**